docs/connections.rst

```diff
@@ -3,3 +3,22 @@ Connections
 
 .. automodule:: nornir_routeros.plugins.connections
    :members:
+
+.. topic:: Usage
+
+   If you need to disable SSL (e.g. during initial provisioning) you need to
+   specify the routerosapi driver under connection options, an example on a
+   host follows this can just as easily be assigned to a group to use across
+   multiple devices:
+
+.. code-block:: yaml
+   :linenos:
+
+   labMT:
+     hostname: 172.16.1.1
+     platform: routeros
+     port: 8728
+     connection_options:
+       routerosapi:
+         extras:
+         use_ssl: False
```

Comment:
```suggestion
           use_ssl: False
```

I believe that this should be indented

Reply:
Ahh, yes you are correct. My bad with missing that.

Its cool with the documentation, I can help as I'm now starting with nornir and can document from the perspective of someone new to nornir. The major thing that threw me off was the routerosapi module definition as everything else is routeros. Figured that out once I looked in plugins/connections/__init__.py .